addons/mail/static/src/discuss/core/web/discuss_core_web_service.js

```diff
@@ -17,6 +17,17 @@ export class DiscussCoreWeb {
     }
 
     setup() {
+        this.busService.subscribe("discuss.channel/joined", async (payload) => {
```

Comment:
Assuming the goal is to remove from `embed`, why only move to `web` and not `web_public`?